### PR TITLE
docs(tutorial/jokes): fix typo

### PR DESCRIPTION
**Pull request by shairez**
Closes: #

- [x] Docs
- [ ] Tests

The `:jokeId` and `$jokeId` convention is confusing. 

The tutorial shows `/jokes/:jokeId` but then talks about `$jokeId` referring to what we just saw.
It looks like a typo

### CODE REVIEW

File: docs/tutorials/jokes.md [hunk 1]
```diff
@@ -400,7 +400,7 @@ Soon we'll add a database that stores our jokes by an ID, so let's add one more
 
 `/jokes/:jokeId`
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:401]: This should be

```suggestion
`/jokes/$jokeId`
```

File: docs/tutorials/jokes.md [hunk 2]
```diff
@@ -400,7 +400,7 @@ Soon we'll add a database that stores our jokes by an ID, so let's add one more
...
 
-Here the parameter `$jokeId` can be anything, and we can lookup that part of the URL up in the database to display the right joke. To make a parameterized route, we use the `$` character in the filename. ([Read more about the convention here][the-route-filename-convention]).
+Here the parameter `:jokeId` can be anything, and we can lookup that part of the URL up in the database to display the right joke. To make a parameterized route, we use the `$` character in the filename. ([Read more about the convention here][the-route-filename-convention]).
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:403]:
```suggestion
Here the parameter `$jokeId` can be anything, and we can lookup that part of the URL up in the database to display the right joke. To make a parameterized route, we use the `$` character in the filename. ([Read more about the convention here][the-route-filename-convention]).
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by MichaelDeBoey** [comment]
@shairez Please sign the CLA in order to get this merged.

**Comment by shairez** [comment]
@MichaelDeBoey  took me a while, but now it's done :)

**Comment by machour** [comment]
Thank you @shairez !

**Pull request merged**